### ISSUE TITLE
Task "Get latest panel release information" delegated to localhost causing errors when Ansible controller host can't become: true

### ISSUE THREAD

**Issue by 15Cyndaquil** [maxhoesel-ansible/ansible-collection-pterodactyl#151]
Is it required to have this task delegated to localhost? Trying to run this in using the docker version [Semaphore ](https://github.com/ansible-semaphore/semaphore) and unfortunately I can't have the entire role included in with `become: true` as the docker container doesn't have sudo installed

```
TASK [maxhoesel.pterodactyl.pterodactyl_panel : Get latest panel release information] ***
fatal: [panel -> localhost]: FAILED! => {"changed": false, "module_stderr": "/bin/sh: sudo: not found\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 127}
```
https://github.com/maxhoesel-ansible/ansible-collection-pterodactyl/blob/0b63dae9c23752813973cc5a859ba5fbb42a331c/roles/pterodactyl_panel/tasks/get_version.yml#L1-L11

And importing the role without `beome: true` I get permission errors when this task is ran
```
TASK [maxhoesel.pterodactyl.pterodactyl_panel : pterodactyl_panel_webroot is present] ***
fatal: [panel]: FAILED! => {"changed": false, "msg": "There was an issue creating /var/www/pterodactyl as requested: [Errno 13] Permission denied: b'/var/www/pterodactyl'", "path": "/var/www/pterodactyl"}
```
https://github.com/maxhoesel-ansible/ansible-collection-pterodactyl/blob/0b63dae9c23752813973cc5a859ba5fbb42a331c/roles/pterodactyl_panel/tasks/main.yml#L13-L19

**Comment by maxhoesel** [comment]
If I'm understanding this correctly, the issue here is that the role is trying to `become` root even during the delegated task - something that's not guaranteed be possible. This should be fixed by just setting `become` to false, I've opened a PR [here](https://github.com/maxhoesel-ansible/ansible-collection-pterodactyl/pull/153). Can you test this PR and verify that it works for you?

You can install this branch like so:

`ansible-galaxy collection install git+https://github.com/maxhoesel-ansible/ansible-collection-pterodactyl.git,delegate_become`

Or in a `requirements.yml`:

```yaml
collections:
  - name: https://github.com/maxhoesel-ansible/ansible-collection-pterodactyl.git
    type: git
    version: "delegate_become"
```

**Comment by 15Cyndaquil** [comment]
Looks like that fixed allows both tasks to go through without issues. Though I am getting the below issue on task past those that I'm not entirely sure what is causing it. 

Though I will say that I have found another automated solution for installed pterodactyl in the the mean time so while I'd be happy to help test fixes for that as well feel free to close this issue

```
TASK [maxhoesel.pterodactyl.pterodactyl_panel : Run composer install] **********
9:48:23 PM
fatal: [test-10]: FAILED! => {"msg": "Failed to set permissions on the temporary files Ansible needs to create when becoming an unprivileged user (rc: 1, err: chmod: invalid mode: ‘A+user:www-data:rx:allow’\nTry 'chmod --help' for more information.\n}). For information on working around this, see https://docs.ansible.com/ansible-core/2.14/user_guide/become.html#risks-of-becoming-an-unprivileged-user"}
```

**Comment by maxhoesel** [comment]
Ah, I'm guessing you're connecting to the remote server as non-root? This happens when Ansible needs to go from a normal user to root (with become: yes), then to a different normal user (such as www-data to install the panel). You can fix it by installing the `acl` package. Should probably add that to the README.

I've merged #153, so this issue should be fixed
